addons/binding/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/internal/convention/homeassistant/AbstractComponent.java

```diff
@@ -41,6 +42,7 @@
 @NonNullByDefault
 public abstract class AbstractComponent<C extends BaseChannelConfiguration> {
     // Component location fields
+    private ComponentConfiguration componentConfiguration;
```

Comment:
Any reason why its not final?

Reply:
Dis you also see this one?

Reply:
Can you merge this PR nevertheless please? There 1-5 more PRs to come and this code will be touched again anyhow.

The initial 3000 LOC PR was presented a month ago now and allows us to be almost 100% MQTT HA compliant and I asked Jochen to split it up into smaller pieces, easier to review for me. 
And he did and his PRs are of really good quality.

There will always be the one small imperfection somewhere, but I will make sure that it is always functional correct and resource efficient.

Please let me do my job as mqtt maintainer and decide when to merge a PR, as I know the greater plan for this module. (It's not actually a critic towards you Martin, but the general process that I observe here, where contributors loose interest, because the patch flow is slowed down for nitpick reasons)

Reply:
@martinvw Yes, I saw it, but I could not fix it while at work.
Not special reason, just a late addition and I forgot to make it final.
I have no problem to change it ;-)

Reply:
Given you responded on the other one I just wanted to make sure that we  could wrap up quick I did not realize more would be commng soon